src/core/components/dropdown/dropdownItem/dropdownItem.tsx

```diff
@@ -1,9 +1,9 @@
 import * as RadixDropdown from '@radix-ui/react-dropdown-menu';
 import classNames from 'classnames';
-import React from 'react';
+import React, { type ComponentProps } from 'react';
 import { Icon, IconType } from '../../icon';
 
-export interface IDropdownItemProps extends Omit<RadixDropdown.DropdownMenuItemProps, 'asChild'> {
+export interface IDropdownItemProps extends Omit<ComponentProps<'div'>, 'dir' | 'onSelect'> {
```

Comment:
Looks like we don't need to omit `dir` props here, right?
```suggestion
export interface IDropdownItemProps extends Omit<ComponentProps<'div'>, 'onSelect'> {
```